lib/src/request.dart

```diff
@@ -113,10 +113,10 @@ class Request extends BaseRequest {
           'content-type "application/x-www-form-urlencoded".');
     }
 
-    return Uri.splitQueryString(body, encoding: encoding);
+    return splitQueryString(body, encoding: encoding);
```

Comment:
Not familiar with term: CL

I've forked the collection package and added my code, do I perform a pull request and reference the original issues?